src/Uno.Sdk/targets/Uno.Implicit.Packages.ProjectSystem.targets

```diff
@@ -0,0 +1,78 @@
+	</ItemGroup>
+
+	<ItemGroup Condition="$(UnoFeatures.Contains(';Material;'))">
+		<PackageReference Include="Uno.Material.WinUI" ProjectSystem="true" />
+		<PackageReference Include="Uno.Material.WinUI.Markup" ProjectSystem="true" Condition="$(UnoFeatures.Contains(';CSharpMarkup;'))" />
+		<PackageReference Include="Uno.Themes.WinUI.Markup" ProjectSystem="true" Condition="$(UnoFeatures.Contains(';CSharpMarkup;'))" />
+	</ItemGroup>
+
+	<ItemGroup Condition="$(UnoFeatures.Contains(';Cupertino;'))">
+		<PackageReference Include="Uno.Cupertino.WinUI" ProjectSystem="true" />
+	</ItemGroup>
+
+	<ItemGroup Condition="$(UnoFeatures.Contains(';Prism;'))">
+		<PackageReference Include="Prism.Uno.WinUI" ProjectSystem="true" Condition="!$(_IsExecutable)" />
+		<PackageReference Include="Prism.DryIoc.Uno.WinUI" ProjectSystem="true" Condition="$(_IsExecutable)" />
+		<PackageReference Include="Prism.Uno.WinUI.Markup" ProjectSystem="true" Condition="$(UnoFeatures.Contains(';CSharpMarkup;'))" />
+	</ItemGroup>
+
+	<ItemGroup Condition="$(UnoFeatures.Contains(';Dsp;'))">
+		<PackageReference Include="Uno.Dsp.Tasks" ProjectSystem="true" />
+	</ItemGroup>
+
+	<ItemGroup Condition="$(UnoFeatures.Contains(';Mvvm;'))">
+		<PackageReference Include="CommunityToolkit.Mvvm" ProjectSystem="true" />
+	</ItemGroup>
+
+	<Import Project="$(MSBuildThisFileDirectory)Uno.Extensions.Implicit.Packages.ProjectSystem.targets" />
+	<Import Project="$(MSBuildThisFileDirectory)Uno.Implicit.Packages.ProjectSystem.Android.targets"
+		Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'android'" />
+	<Import Project="$(MSBuildThisFileDirectory)Uno.Implicit.Packages.ProjectSystem.Desktop.targets"
+		Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'desktop'" />
+	<Import Project="$(MSBuildThisFileDirectory)Uno.Implicit.Packages.ProjectSystem.iOS.targets"
+		Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'ios'" />
+	<Import Project="$(MSBuildThisFileDirectory)Uno.Implicit.Packages.ProjectSystem.MacCatalyst.targets"
+		Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'maccatalyst'" />
+	<Import Project="$(MSBuildThisFileDirectory)Uno.Implicit.Packages.ProjectSystem.MacOS.targets"
+		Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'macos'" />
+	<Import Project="$(MSBuildThisFileDirectory)Uno.Implicit.Packages.ProjectSystem.Wasm.targets"
+		Condition=" '$(IsBrowserWasm)' == 'true' " />
+	<Import Project="$(MSBuildThisFileDirectory)Uno.Implicit.Packages.ProjectSystem.Desktop.targets"
+		Condition="$(TargetFramework.Contains('windows10'))" />
+
+	<Target Name="UnoProjectSystemReferenceCleanup"
+		BeforeTargets="UnoImplicitPackages">
+		<ItemGroup>
+			<PackageReference Remove="@(PackageReference->HasMetaData('ProjectSystem'))" />
```

Comment:
```suggestion
                        <!-- 
                           Remove all placeholder references so that implicit references can be created with
                           actual versions using the `UnoImplicitPackages` target.
                        -->
			<PackageReference Remove="@(PackageReference->HasMetaData('ProjectSystem'))" />
```